### PR TITLE
Document EOL support dates of test platforms

### PR DESCRIPTION
**Pull request by NotMyFault**
Let's write them down here, so we don't need to look them up now and then.

### CODE REVIEW

File: molecule/default/molecule.yml [hunk 1]
```diff
@@ -6,39 +6,39 @@ driver:
...
+  - name: debian-11 # EOL 2026-06-30
     image: dokken/debian-11:latest
     override_command: false
     volumes:
       - ${MOLECULE_PROJECT_DIRECTORY}/target:/var/tmp/target
       - /sys/fs/cgroup:/sys/fs/cgroup:rw
     cgroupns_mode: host
     privileged: true
-  - name: ubuntu-18-04
+  - name: ubuntu-18-04 # # EOL 2023-05-31
```
Comment (hervelemeur) [inline molecule/default/molecule.yml:25]:
```suggestion
  - name: ubuntu-18-04 # EOL 2023-05-31
```

Comment (NotMyFault) [reply molecule/default/molecule.yml]: Oh, good catch 👍

**Review by basil — COMMENTED** [review]
This change introduces `yamllint(1)` errors:

```
$ yamllint molecule/default/molecule.yml
molecule.yml
  9:21      warning  too few spaces before comment  (comments)
  17:21     warning  too few spaces before comment  (comments)
  25:24     warning  too few spaces before comment  (comments)
  33:24     warning  too few spaces before comment  (comments)
  41:24     warning  too few spaces before comment  (comments)
  50:23     warning  too few spaces before comment  (comments)
  58:23     warning  too few spaces before comment  (comments)
  66:25     warning  too few spaces before comment  (comments)
  74:27     warning  too few spaces before comment  (comments)
  82:27     warning  too few spaces before comment  (comments)
  90:21     warning  too few spaces before comment  (comments)
  98:21     warning  too few spaces before comment  (comments)
  106:25    warning  too few spaces before comment  (comments)
  114:25    warning  too few spaces before comment  (comments)
  122:24    warning  too few spaces before comment  (comments)
  130:24    warning  too few spaces before comment  (comments)
  139:28    warning  too few spaces before comment  (comments)
```

I would suggest just adding another space before the pound sign for each comment.

**Comment by NotMyFault** [comment]
> This change introduces yamllint(1) errors:

There we go, applied yamllint fixes for the diff. Do we actually care about yamllint? I ran it on the molecule directory and can locate a handful number of errors.

**Comment by basil** [comment]
> Do we actually care about yamllint?

I certainly care about its indentation and spacing warnings. I do not care about its line length warnings. The only non line length warnings in the existing code are in `docker-compose.yaml`. Since I did not write that file, I do not feel obligated to fix the existing warnings.

**Review by basil — APPROVED** [review]

**Pull request merged**